src/main/resources/conf/chains/eth-chain.conf

```diff
@@ -98,6 +98,10 @@
   # https://ecips.ethereumclassic.org/ECIPs/ecip-1099
   ecip1099-block-number = "1000000000000000000"
 
+  # Magneto EVM and Protocol Upgrades
+  # https://ecips.ethereumclassic.org/ECIPs/ecip-1103
+  magneto-block-number = "1000000000000000000"
```

Comment:
Instead of using a super high block number, why not define this config value as `Option[BigInt]`, and omit it in the files where it does not make sense?

Reply:
I was thinking about doing it like this, but in the end didn't to remain consistent. Perhaps it would make sense to refactor those configs with the approach above in a separate PR?

Reply:
There is a configuration added by me, this an `Option` already :) So total consistency is already gone 😅 
I would love to see this refactored